lib/modules/pipeline/index.js

```diff
@@ -1,64 +1,60 @@
-const fs = require('../core/fs.js');
+const fs = require('../../core/fs.js');
 const async = require('async');
-const ProcessLauncher = require('../core/processes/processLauncher');
-const utils = require('../utils/utils.js');
-const constants = require('../constants');
+const utils = require('../../utils/utils.js');
+const ProcessLauncher = require('../../core/processes/processLauncher');
```

Comment:
We probably want to make this one a plugin module as well. Looping in @iurimatias for feedback.